### DISCUSSION TITLE
arttwine - A community sourced platform for combining and downloading art.

### DISCUSSION THREAD

**Discussion by AkshayTati** [scaleracademy/open-source-projects#28] [Ideas]
### Summary

The platform is supposed to be a place  where people can upload art for others to view, combine and download it for their own use. (arttwine = art + intertwine)

Not all developers are designers and it can get difficult at times to find good art for your website/app or whatever you're building, and its even harder if you want the art to be original, this platform is supposed to make that task easier. Lets take an example use case -
       
_You want a specific globohomo art design for your app. You go the website, pick up any globohomo head, combine it with a body, hands, legs. You've got your character ready, next you want a table, you look up a table and add any table to your design. Then you can add a laptop on the table. Easy as that! Then the user can download a transparent image of that for them to use._

### Functionality 
1. Everytime a user downloads something they get an advertisement
2. Revenue generated is used to keep the website and community running
3. The original artists get some incentive based on how many times their art has been used

### Features
1. Weekly/Monthly Leaderboards for the most popular art.
2. Weekly/Monthly Leaderboards for the most popular artists.
3. Navbar has User profile, Leaderboards, Search Bar, categories for art.
4. User Profile - 
    -  Your concoctions
    -  Your contributions
    -  Revenue(if any)

### User Stories
1. Step 1 - Login/Sign up page for creating an account.
2. Step 2 - Two Sides to go into - 1.Contribute or 2.Combine
    1.Contribute - Users upload their own original art for others to use.
     2.Combine  - Users combine and download art for themselves.

### Future Scope

- Integrating an entire design panel for creating art in the website itself.
- Collecting data on what type of art is more popular in which domains, which art gets more clicks, views.
- Integration with other designing apps/websites.

**Comment by championswimmer** [comment]
https://blush.design/ does a bit of this right ?

**Reply by AkshayTati** [reply]
Oh well yes that does seem similar, I didn't know about this site. 
I checked it out, I guess the difference boils down to two things -

- Blush is mostly not free, many things are locked by a pay wall
- It isn't community sourced, the designs are made by a few select artists which limits its catalogue(still huge tho).
